pkgs/development/python-modules/python-fluent/fluent-syntax.nix

```
@@ -0,0 +1,38 @@
+{ buildPythonPackage, fetchFromGitHub, lib, typing-extensions }:
+
+let
+  version = "0.19.0";
+  tag = "fluent.syntax@${version}";
+
+  src = fetchFromGitHub {
+    owner = "projectfluent";
+    repo = "python-fluent";
+    rev = tag;
+    hash = "sha256-nULngwBG/ebICRDi6HMHBdT+r/oq6tbDL7C1iMZpMsA=";
+  };
+in
+buildPythonPackage {
+  pname = "fluent-syntax";
+  inherit version;
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
also make sure to add setuptools to build-dependencies

---

As mentioned in a previous review comment, the upstream project doesn't have a `pyproject.toml` in any of its latest release tags. Would this change be needed still?

---

That is not necessary, the builder will fallback to setuptools and we want to use pyproject = true in the future for all packages possible. Please see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#:~:text=in%20wrapped%20programs.-,pyproject,-%3A%20Whether%20the%20pyproject